Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/DefaultDartFeedbackBuilder.java

```diff
@@ -3,25 +3,51 @@
 import com.intellij.ide.BrowserUtil;
 import com.intellij.openapi.application.ex.ApplicationInfoEx;
 import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.io.FileUtil;
 import com.jetbrains.lang.dart.DartBundle;
 import com.jetbrains.lang.dart.sdk.DartSdk;
 import org.jetbrains.annotations.Nullable;
 
+import java.io.File;
+import java.io.IOException;
+
 import static com.intellij.ide.actions.SendFeedbackAction.getDescription;
 
 public class DefaultDartFeedbackBuilder implements DartFeedbackBuilder {
+  private String myMessage;
 
   public String prompt() {
     return "Create issue on github?";
   }
 
+  public void setMessage(String message) {
+    myMessage = message;
+  }
+
   public void sendFeedback(@Nullable Project project) {
     final ApplicationInfoEx appInfo = ApplicationInfoEx.getInstanceEx();
     boolean eap = appInfo.isEAP();
     String ijBuild = eap ? appInfo.getBuild().asStringWithoutProductCode() : appInfo.getBuild().asString();
     String sdkVsn = getSdkVersion(project);
-    String platDescr = getDescription();
+    String platDescr = getDescription().replaceAll(";", " ");
     String template = DartBundle.message("dart.feedback.url.template", ijBuild, sdkVsn, platDescr);
+    if (myMessage != null) {
+      myMessage = "```dart\n" + myMessage + "```";
```

Comment:
Will it modify myMessage several times if clicked on 'Submit feedback' button several times?